locales/zh_CN/LC_MESSAGES/duckduckgo.po

```diff
@@ -760,7 +760,7 @@ msgstr "清除"
 
 msgctxt "precise_user_location"
 msgid "Clear"
-msgstr ""
+msgstr "清除"
 
 msgid "Clear All"
 msgstr "清除"
```

Comment:
Yes, that makes a lot of sense. But from the context I’m afraid it’s still somewhat vague about what exactly is cleared. Maybe 清空历史 would be more intuitive?
![](https://user-images.githubusercontent.com/48396683/83769377-580f2b00-a6b2-11ea-8503-d006780a8260.png)

Reply:
That's a good point. I suppose a fuller English version would be "Clear all regions" or "Clear region history". I'm a bit reluctant to use 清空历史 because some people might think it means clear search history, but as you know we don't keep any personal search history.

Perhaps 清空地区历史 would work instead. I did a quick mock-up and it seems to fit, but if you think it's better to keep it short then we can stick with just 清空. 

![Screen Shot 2020-06-05 at 15 20 08](https://user-images.githubusercontent.com/94173/83843536-26dd3c00-a740-11ea-9fed-4568b5604f3b.png)

Reply:
On second thoughts, maybe 清空地区, which is already used in the .po file?

Reply:
清空地区 might not be specific enough. And compared with 清空地区历史, I found 清空常用地 a good alternative, leaving some more space for the search box.
![](https://user-images.githubusercontent.com/48396683/83872536-c4e2fd80-a764-11ea-8ad8-f59a47847851.png)
By the way, as can be seen in the screenshot above, the country filter doesn’t seem to work when accessed from China. It simply gives the same results, mostly in Chinese, regardless of the country and UI language settings. Do you have any ideas what’s going on?

Reply:
That's a good idea – I like 清空常用地. Would you be able to update the PR to use that term please?

As for the country filter not working for you, do you get the same result if you change the UI language (in Settings) to English?

Reply:
Sure. I’ve commited it.
> do you get the same result if you change the UI language (in Settings) to English?

Yes.
![](https://user-images.githubusercontent.com/48396683/83879391-20ff4f00-a770-11ea-82e9-ccfba6e12814.png)

Reply:
Great, thank you for your patience with this, and for helping out Chinese-speaking users.

And I'll file the bug about the results not being from the selected region. I'm not sure what's happening there. Anyway, 谢谢!

Reply:
OK. Many thanks for the assistance.

Reply:
Hi again @b9a1. Sorry to come back to this but could you check whether the Chinese/English results issue also happens in Bing or Ecosia? As you may know, we get most of our results from Bing so this would help us know where the problem's happening.

Reply:
Yes, and it seems to be quite relevant. Unlike in other countries, Bing features two editions in Mainland China: a ‘domestic edition’ that prefers Chinese results and an ‘international edition’ that does not. You should be able to see what it looks like via [`https://www.bing.com/?mkt=zh-CN`](https://www.bing.com/?mkt=zh-CN). And as you might have guessed, the usual region option in the settings page has no effect and is always set to China, so the results stay the same just like in DDG. Hope this helps.

Reply:
Hello again. This is very helpful. I didn't realise there were domestic and international editions of Bing. I've forwarded this information to our developers and hopefully we can make improvements. Thank you.